src/openrct2/ride/Vehicle.cpp

```diff
@@ -7952,7 +7952,8 @@ static bool vehicle_update_track_motion_forwards_get_new_track(
                         SoundId::BlockBrakeRelease, { vehicle->track_x, vehicle->track_y, vehicle->track_z });
                 }
             }
-            map_invalidate_element(vehicle->track_x, vehicle->track_z, tileElement);
+            // This is very weird because it uses Z
```

Comment:
This is a very weird comment :P 

Seriously though: what is weird about it using Z?

Reply:
I meant that all other calls use `x,y` but this uses `x,z`

Reply:
Could it be a refactoring typo? IIRC, Duncan recently refactored this, it might have been using y previously.

Reply:
It seems like it used to be `x, y`, didn't go as far as finding out who changed, but I fixed it.